src/darwin/CHIPTool/CHIPTool/Framework Helpers/CHIPConnectivityManager.h

```diff
@@ -19,18 +19,8 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@protocol CHIPConnectivityManagerDelegate;
-
 @interface CHIPConnectivityManager : NSObject
 + (CHIPConnectivityManager *)sharedManager;
```

Comment:
can we just remove this file and its .m 
They do not seem to be useful anymore

Reply:
Sure, makes sense. I had confused `sharedManager` with `sharedController`, and assumed that we needed it.